contrib/python-filelock/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "python-filelock"
+pkgver = "3.13.1"
+pkgrel = 0
+build_style = "python_pep517"
+_deps = [
+    "python-hatchling",
+    "python-hatch_vcs",
+]
```

Comment:
these are neither checkdepends nor runtime depends so they should be in hostmakedepends only

Reply:
fixed now

Reply:
(you can remove the whole _deps thing and put them in hostmakedepends since it's only used in that one place now)

Reply:
of course; done